credentials/credentials.go

```diff
@@ -100,13 +120,17 @@ func Get(helper Helper, reader io.Reader, writer io.Writer) error {
 	}
 
 	serverURL := strings.TrimSpace(buffer.String())
+	if len(serverURL) == 0 {
```

Comment:
instead of this individual check, should we instead call `resp.isValid()` after forming it below?  That way we can also verify the username